concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -247,6 +250,11 @@ resources:
     source:
       interval: 5m
 
+  - name: smtp-credential-rotation-timer
```

Comment:
I would challenge that we rotate the credentials in timely manner. That would force the generation of a new manifest every 720h, and then force a restart of UAA. 

If the requierement is not keep permanent AWS keys in the state file, we can use the property https://www.terraform.io/docs/providers/aws/r/iam_access_key.html#pgp_key by passing a random pregenerated GPG public key for which we do not keep the private key anyware.

By encrypting the AWS access key with a gpg key without private key, we are effectively not keeping those keys at all! The SES credentials would be accessible anyway.

Reply:
We don't care that the keys are in the state file - that's okay. Deleting the key from the state file is a hack that causes Terraform to generate a new key on the next run. All unused keys will be cleared up after every deploy.

Reply:
But what is the motivation of generating new keys every X hours?